one_fm/api/tasks.py

```diff
@@ -863,6 +890,13 @@ def calculate_penalty_amount(employee, start_date, end_date, logs):
 		'docstatus': 1,
 		'employee': employee
 	}
+	if frappe.db.exists('Employee', {'employee':employee, 'status':'Left'})):

```

Comment:
@pjamsheer , there is a fix over here. Can you remove the extra bracket at the end?